Dockerfile-alpine.template

```
@@ -52,7 +56,14 @@ ENV PG_MAJOR {{ env.version }}
 ENV PG_VERSION {{ .version }}
 ENV PG_SHA256 {{ .sha256 }}
 
-{{ def llvmver: "15" -}}
+{{
+	def llvmver:
+		if alpine_version | split(".")[1] | tonumber < 21 then
```

I don't like the implicit `3.x` assumption here :sweat_smile:  Maybe something like this?

```suggestion
		if alpine_version | split(".") | map(tonumber) < [3, 21] then
```

```console
$ jq -rnc '
	"2.22", "3.20", "3.21", "3.22", "4.0",
	"2.22.1", "3.20.1", "3.21.1", "3.22.1", "4.0.1"
	| "\(.): \(split(".") | map(tonumber) < [3, 21])"
'
2.22: true
3.20: true
3.21: false
3.22: false
4.0: false
2.22.1: true
3.20.1: true
3.21.1: false
3.22.1: false
4.0.1: false
```

---

Yeah, that's better.